README.md

```diff
@@ -39,6 +39,9 @@ more deeply nested.  It would be even more difficult to read.
 
 ![example screenshot](https://raw.githubusercontent.com/cdepillabout/pretty-simple/master/img/pretty-simple-example-screenshot.png)
 
+There's a [web app](https://cdepillabout.github.io/pretty-simple) compiled with
+GHCJS where you can play around with `pretty-simple` running in your browser.
```

Comment:
@georgefst I added this to the README, but please feel free to send a follow up PR or something if you think there is a better way to show it off.  I think it is really neat, and I bet potential users might really want to play with it.

It is also a useful tool in its own right, similar to the `pretty-simple` CLI tool.

Reply:
Cool, yeah I've been meaning to do this, as well as adding a link back to Hackage/GitHub from the web app.